Objective-C/Tests/URLEndpointListenerTest.m

```diff
@@ -0,0 +1,162 @@
+//
+//  URLEndpointListenerTest.m
+//  CouchbaseLite
+//
+//  Copyright (c) 2020 Couchbase, Inc All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import "ReplicatorTest.h"
+#import "CBLURLEndpointListener.h"
+#import "CBLURLEndpointListenerConfiguration.h"
+#import "CollectionUtils.h"
+
+#define kPort 4984
+#define kURL [NSURL URLWithString: $sprintf(@"ws://localhost:%d/otherdb", kPort)]
+
+API_AVAILABLE(macos(10.12), ios(10.0))
+@interface URLEndpointListenerTest : ReplicatorTest
+typedef CBLURLEndpointListenerConfiguration Config;
+typedef CBLURLEndpointListener Listener;
+
+@end
+
+@implementation URLEndpointListenerTest
+
+- (Listener*) listenAt: (uint16)port {
+    Config* config = [[Config alloc] initWithDatabase: self.otherDB port: port];
+    config.disableTLS = YES;
+    Listener* list = [[Listener alloc] initWithConfig: config];
```

Comment:
I feel like list is not a good variable name for this as list also means List. Can we change it to listener?